pkg/cmd/pulumi/import.go

```diff
@@ -457,7 +457,7 @@ func newImportCmd() *cobra.Command {
 			var programGenerator programGeneratorFunc
 			switch proj.Runtime.Name() {
 			case "dotnet":
-				programGenerator = dotnet.GenerateProgram
+				programGenerator = dotnet.GenerateProgramForImport
```

Comment:
I know that I requested this, but after [the comment from @mikhailshilkov](https://github.com/pulumi/pulumi/pull/10143#issuecomment-1185314371), I think we can drop it back to `dotnet.GenerateProgram`, removing `dotnet.GenerateProgramForImport`.